samples/cpp/tutorial_code/dnn/text_spotting/text_spotting.cpp

```diff
@@ -0,0 +1,82 @@
+#include <iostream>
```

Comment:
Ad for me, too much samples with the same meaning - high level API should be as flexible as possible

Reply:
I have deleted the samples in samples/cpp/tutorial_code/dnn/text_spotting, and only embed snippets in the tutorial.